param/__init__.py

```diff
@@ -27,9 +27,10 @@
 from .parameterized import Parameterized, Parameter, String, \
      descendents, ParameterizedFunction, ParamOverrides
 
-from .parameterized import depends, output   # noqa: api import
-from .parameterized import logging_level     # noqa: api import
-from .parameterized import shared_parameters # noqa: api import
+from .parameterized import depends, output     # noqa: api import
+from .parameterized import logging_level       # noqa: api import
+from .parameterized import shared_parameters   # noqa: api import
+from .parameterized import instance_descriptor # noqa: api import
```

Comment:
Seems like a strange mix of formatting styles; I'd favor matching the style on on line 27 to avoid all the redundancy.